config/config.go

```diff
@@ -109,6 +109,17 @@ var (
 	DefaultMarathonSDConfig = MarathonSDConfig{
 		RefreshInterval: Duration(30 * time.Second),
 	}
+
+	// The default Kubernetes SD configuration
+	DefaultKubernetesSDConfig = KubernetesSDConfig{
+		NodeMetricsPort: 10255,
+		ScrapeKey:       "prometheus.io/scrape",
```

Comment:
What is the difference in doing this via relabelling or via SD config? Not sure. This isn't specific to my setup, but a pre-configured key to be used on annotations on service to enable scraping of service endpoints. I'm totally unsure how relabelling could be used in this case as this annotation actually prevents target groups from being created if the key isn't present & truthy.